src/ansys/fluent/core/launcher/launcher_utils.py

```diff
@@ -347,7 +359,10 @@ def _await_fluent_launch(
             raise TimeoutError("The launch process has timed out.")
         time.sleep(1)
         start_timeout -= 1
-        logger.info(f"Waiting for Fluent to launch...{start_timeout} seconds remaining")
+        if start_timeout >= 0:
+            logger.info(
+                f"Waiting for Fluent to launch...{start_timeout} seconds remaining"
+            )
```

Comment:
Minor suggestion, I think it will be helpful to have a logging call indicating that PyFluent is now going to be waiting for Fluent to launch, regardless of timeout configuration (particularly now that we will support waiting indefinitely).
```suggestion
        logger.info(f"Waiting for Fluent to launch...")
        if start_timeout >= 0:
            logger.info(
                f"...{start_timeout} seconds remaining"
            )
```